### ISSUE TITLE
updates

### ISSUE THREAD

**Issue by m-bosch** [grantemsley/MultiPoolMiner#26]
Hi Grant,

will this still be updated? I see several updates in main multipoolminer and they are also have a pre release for new version.

Can I stay at your repo and get updated or will this be closed?

**Comment by grantemsley** [comment]
No, all my changes have been incorporated into the official one, so you should switch to using that.

**Comment by m-bosch** [comment]
Hi grant,


So the beta 3.1 has your updater etc in it?

**Comment by m-bosch** [comment]
I don't see the launcher and updater in that version but if you advise us to go back to that I will do.
